app/src/composables/useTabCoordinator.ts

```
@@ -0,0 +1,304 @@
+
+  function setInCall(value: boolean) {
+    _inCall = value;
+  }
+
+  function becomeLeader() {
+    if (isLeader.value) return;
+    isLeader.value = true;
+    currentLeaderTabId = tabId;
+    otherTabInCall.value = false;
+    startLeaderHeartbeat();
+    onBecomeLeader.forEach((cb) => cb());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)